packages/perseus-editor/src/components/json-editor.jsx

```diff
@@ -1,4 +1,7 @@
-/* eslint-disable @babel/no-invalid-this, react/no-unsafe, react/sort-comp */
+/* eslint-disable react/prop-types */
+/* eslint-disable @babel/no-invalid-this */
+/* eslint-disable react/no-unsafe */
+/* eslint-disable react/sort-comp */
```

Comment:
Agree that I like each disable on its own line. Diffs that change which are ignored are much clearer!